microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md

```diff
@@ -48,90 +47,94 @@ See also:
 * [Analyze selector performance during Recalculate Style events](../../../evaluate-performance/selector-stats.md)
 * [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)
 
+
 <!-- ====================================================================== -->
-## The Issues tool warns you when CSS properties trigger reflow 
+## The Issues tool warns you when CSS properties trigger reflow
+<!-- todo: is this a change to the Issues tool, Styles pane, or both? -->
```

Comment:
Change the title to say "Styles pane" instead of "Issues tool"?  
Is this a change to the Issues tool, Styles pane, or both?  The main announcement text focuses on Styles pane, not Issues tool, as being new behavior:

Now, the **Styles** pane in the **Elements** tool adds a wavy underline on CSS properties that trigger reflow.

Reply:
it is both.

My mental model is that the main announcement is 

"Devtools now detects and warns on properties that can cause performance issues when animated"

We raise it in two ways:
- On the `Issues Panel` under `Performance` category you will see a new issue being reported e.g
    "'left' changes to this property will trigger: 'Layout', which can impact performance when used inside @keyframes."
- On the `Styles Pane` we add wavy underline on CSS properties that trigger reflow.

Reply:
@zoherghadyali @jenna-sasson - Change the title or Subtitle?  The title only mentions Issues tool.  Subtitle only mentions Styles pane.

rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114?branch=pr-en-us-2618#the-issues-tool-warns-you-when-css-properties-trigger-reflow
[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/whats-new-114/microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md#the-issues-tool-warns-you-when-css-properties-trigger-reflow)

Reply:
rewrote heading & Subtitle, see latest Files Changed tab or rendered page.